include/quda.h

```diff
@@ -732,6 +732,12 @@ extern "C" {
     /** Boolean for aggregation type, implies staggered or not */
     QudaTransferType transfer_type[QUDA_MAX_MG_LEVEL];
 
+    /** Whether or not to let MG coarsening drop improvements, for ex dropping long links in small aggregation dimensions */
```

Comment:
These variables need to propagated into quda_fortran.F90

Reply:
Not to make excuses, but the multigrid structure doesn't seem to be included in `quda_fortran.F90` anyway?

Reply:
Resolved offline